UPGRADE-3.0.md

```diff
@@ -0,0 +1,95 @@
+# Upgrading from 2.x to 3.0
+
+The main backward compatibility break in lmc/coding-standard version 3.0 is new PHP format used for
+code-style configuration. However, the changes are mostly "mechanical" and most of them can be automated.
+
+### 1. Update dependency in composer.json
+In require-dev section change the version constraint:
+
+```diff
+-        "lmc/coding-standard": "^2.1",
++        "lmc/coding-standard": "^3.0",
```

Comment:
If you want to use it already until the 3.0 version is actually released, one must obviously use branch constrain.